getting-started/deployment.md

```diff
@@ -6,6 +6,10 @@ Deploying a Solidus application to production is no different from deploying any
 
 ### Heroku
 
+{% hint style="warning" %}
+Heroku [does not support](https://devcenter.heroku.com/articles/sqlite3) the Rails default database Sqlite3. Heroku natively utilizes [PostgreSQL](https://www.postgresql.org/) and adaptively uses other [data storage services](https://elements.heroku.com/addons).
```

Comment:
I think we should point out how to use Sqlite3 in development and Postgres in production.